cmd/rp-archiver/main.go

```diff
@@ -79,6 +79,30 @@ func main() {
 		logrus.WithError(err).Fatal("cannot write to temp directory")
 	}
 
+	semaphore := make(chan struct{}, config.MaxConcurrentArchivation)
+
+	archiveTask := func(org archives.Org) {
+		defer func() { <-semaphore }()
+		// no single org should take more than 12 hours
+		ctx, cancel := context.WithTimeout(context.Background(), time.Hour*12)
```

Comment:
What about letting the time be an environment variable? If we see in the future that 12 is a long time, we can change it easily.